images/virtualization-artifact/pkg/controller/vd/internal/attachee.go

```diff
@@ -52,7 +67,22 @@ func (h AttacheeHandler) Handle(ctx context.Context, vd *virtv2.VirtualDisk) (re
 		vd.Status.AttachedToVirtualMachines = append(vd.Status.AttachedToVirtualMachines, virtv2.AttachedVirtualMachine{
 			Name: vm.GetName(),
 		})
+
+		if vm.Status.Phase == virtv2.MachineRunning {
```

Comment:
May be it's better to check the Running condition here. Migrating phase is also Running state of a virtual machine

Reply:
Fixed.